packages/core/src/lib/loggers/httpLogger.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  BaseActivityLogger,
+  ActivityLoggerType,
+} from '../../models/extensions/logger';
+import {
+  VulcanExtensionId,
+  VulcanInternalExtension,
+} from '../../models/extensions';
+import axios, { AxiosRequestHeaders } from 'axios';
+import { ConnectionConfig } from '../utils/url';
+
+export interface HttpLoggerConfig {
+  connection?: HttpLoggerConnectionConfig;
+}
+
+export interface HttpLoggerConnectionConfig extends ConnectionConfig {
+  headers?: Record<string, string | number | boolean> | undefined;
+}
+
+@VulcanInternalExtension('activity-log')
+@VulcanExtensionId(ActivityLoggerType.HTTP_LOGGER)
+export class HttpLogger extends BaseActivityLogger<HttpLoggerConfig> {
+  private logger = this.getLogger();
+
+  public async log(payload: any): Promise<void> {
+    if (!this.isEnabled()) return;
+    const option = this.getOptions();
+    if (!option?.connection) {
+      throw new Error('Http logger connection should be provided');
+    }
+    const headers = option.connection.headers;
+    const url = this.getUrl(option.connection);
```

Comment:
Just curious seems you have provided a function called `getUrl` in `utils` and here you also create the same name method, but also the same code ( not using the `header` ), so why does not use the `getUrl` in `utils` directly? 

```
const url = getUrl(connection as ConnectionConfig);
```